src/Utils.js

```diff
@@ -157,11 +158,11 @@ export function coercePropValue(propValue) {
 
   // if propValue includes quotes, it should be
   // treated as a string
-  if (propValue.search(/"/) !== -1) {
-    return propValue.replace(/"/g, '');
+  if (/^(['"]).*\1$/.test(propValue.trim())) {
```

Comment:
Alternatively to caching, we could do this by modifying the regex a little.

```js
const matches = propValue.match(/^\s*(['"])(.*)\1\s*$);
if (matches) {
  return matches[2];
}
```

or something like that